recipes/mbedtls/all/conanfile.py

```diff
@@ -53,22 +57,19 @@ def configure(self):
             raise ConanInvalidConfiguration("{}-{} is not supported by this recipe".format(self.settings.compiler, self.settings.compiler.version))
```

Comment:
you can also take advantage of this PR to move `ConanInvalidConfiguration` logic in `validate()`

Reply:
Done